msteams-platform/bots/how-to/bot-messages-ai-generated-content.md

```diff
@@ -175,29 +175,48 @@ export const app = new Application<ApplicationTurnState>({
     },
 ```
 
-For more information, see the [const app variable](https://github.com/microsoft/teams-ai/blob/main/js/samples/04.ai-apps/h.datasource-azureOpenAI/src/app.ts#L42).
+For more information, see the [const app variable](https://github.com/microsoft/teams-ai/blob/main/js/samples/04.ai-apps/h.datasource-azureOpenAI/src/app.ts#L65).
 
-After you enable feedback buttons, all `SAY` commands from the bot have `feedbackLoopEnabled` automatically set to `true` in the `channelData` object.
+After you enable feedback buttons, all `SAY` commands from the bot have `feedbackLoopEnabled` in the `channelData` object automatically set to `true`.
 
-To enable feedback buttons in a bot built using **Bot Framework SDK**, add a `channelData` object in your bot message and set the value of `feedbackLoopEnabled` to `true`.
+To enable feedback buttons in a bot built using **Bot Framework SDK**, define a `feedbackLoop` object under the `channelData` object of your bot message.
 
 ```javascript
 await context.sendActivity({
   type: ActivityTypes.Message,
   text: `Hey! I'm a friendly AI bot!`,
   channelData: {
-    feedbackLoopEnabled: true // Enable feedback buttons
+    feedbackLoop: { // Enable feedback buttons
+        type: "custom"
   },
 });
 ```
 
 | Property | Type | Required | Description |
 |--|--|--|--|
-| `feedbackLoopEnabled` | Boolean | Yes | Enables feedback buttons in the bot's message |
+| `feedbackLoop` | Object | ✔️ | Enables feedback buttons in the bot's message. |
+| `feedbackLoop.type` | String | ✔️ | Defines the type of feedback form that appears when a user selects the feedback buttons.<br>Allowed values: `custom`, `default` |
+
+If you set `feedbackLoop.type` to `default`, the default feedback form appears when a user selects the feedback buttons. If you set `feedbackLoop.type` to `custom`, the following invoke request is sent to the bot to retrieve a custom form to be displayed to the user:
+
+```JSON
```

Comment:
Do we have sample code for this? If not, please reach out to sample code team.

Reply:
Yes, the sample code is being created.